optimization/@matRad_OptimizationProblem/matRad_OptimizationProblem.m

```diff
@@ -24,17 +24,27 @@
     
     properties
         BP
+        normalizationScheme = struct('type','none');
+        objectives = {}; %cell array storing all objectives, has to be initialized at the start
+        constraints = {}; %
+        objidx;
+        constridx;
         quantityOpt = '';
         useMaxApprox = 'logsumexp'; %'pnorm'; %'logsumexp'; %'none';
         p = 30; %Can be chosen larger (closer to maximum) or smaller (closer to mean). Only tested 20 >= p >= 1
 
         minimumW = NaN;
         maximumW = NaN;
     end
+
     
     methods
-        function obj = matRad_OptimizationProblem(backProjection)
-            obj.BP = backProjection;
+        function obj = matRad_OptimizationProblem(backProjection,cst)
```

Comment:
I am not sure if we should add the cst as a parameter (compatibility to other stuff).
We could also generate this cache the first time we need it instead?

Reply:
But actually I am not sure what's the best strategy here.... maybe leave it like this.